anecdote/assets/logs/Anecdote Log Pipeline_test_tests.yaml

```diff
@@ -0,0 +1,139 @@
+id: anecdote
+tests:
+  - sample: >-
```

Comment:
the samples provided look to be ingested by Datadog already. The sample logs should be raw logs before they are sent to Datadog.